Gemfile.lock

```diff
@@ -993,4 +993,4 @@ RUBY VERSION
    ruby 2.6.6p146
 
 BUNDLED WITH
-   1.17.3
+   1.17.3
```

Comment:
Could you add the newline back here, or is this something that rubocop auto corrected?

Reply:
I don't know what happened here...github was being weird.  But it is fixed now.